airbyte-cdk/python/unit_tests/sources/file_based/config/test_file_based_stream_config.py

```diff
@@ -5,14 +5,15 @@
 from typing import Any, Mapping, Type
 
 import pytest as pytest
-from airbyte_cdk.sources.file_based.config.file_based_stream_config import CsvFormat, FileBasedStreamConfig, QuotingBehavior
+from airbyte_cdk.sources.file_based.config.csv_format import QuotingBehavior
+from airbyte_cdk.sources.file_based.config.file_based_stream_config import CsvFormat, FileBasedStreamConfig
 from pydantic import ValidationError
 
 
 @pytest.mark.parametrize(
     "file_type, input_format, expected_format, expected_error",
     [
-        pytest.param("csv", {"filetype": "csv", "delimiter": "d", "quote_char": "q", "escape_char": "e", "encoding": "ascii", "double_quote": True, "quoting_behavior": "Quote All"}, {"delimiter": "d", "quote_char": "q", "escape_char": "e", "encoding": "ascii", "double_quote": True, "quoting_behavior": QuotingBehavior.QUOTE_ALL}, None, id="test_valid_format"),
+        pytest.param("csv", {"filetype": "csv", "delimiter": "d", "quote_char": "q", "escape_char": "e", "encoding": "ascii", "double_quote": True, "quoting_behavior": "Quote All"}, {"filetype": "csv", "delimiter": "d", "quote_char": "q", "escape_char": "e", "encoding": "ascii", "double_quote": True, "quoting_behavior": QuotingBehavior.QUOTE_ALL}, None, id="test_valid_format"),
```

Comment:
the file_format object needs the filetype too to so jsonschema knows what type it is